### PR TITLE
TvOS sample app

### PR DESCRIPTION
**Pull request by Matthew-C**
Just ported BasicPlaybackSampleApp to TvOS.

### CODE REVIEW

File: BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -14,14 +15,14 @@
                         <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
                         <prototypes>
                             <tableViewCell contentMode="scaleToFill" selectionStyle="blue" hidesAccessoryWhenEditing="NO" indentationLevel="1" indentationWidth="0.0" reuseIdentifier="Cell" textLabel="Arm-wq-HPj" style="IBUITableViewCellStyleDefault" id="WCw-Qf-5nD">
-                                <rect key="frame" x="0.0" y="86" width="320" height="44"/>
+                                <rect key="frame" x="0.0" y="86" width="600" height="44"/>
                                 <autoresizingMask key="autoresizingMask"/>
                                 <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="WCw-Qf-5nD" id="37f-cq-3Eg">
-                                    <rect key="frame" x="0.0" y="0.0" width="320" height="43"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="600" height="43.5"/>
```
Comment (lental) [inline BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard:21]: is this just an artifact of opening the BasicPlayback main.storyboard, or did you actually modify something?

**Comment by lental** [comment]
can you specifically explain what makes this app different than the BasicPlaybackSampleApp?

File: BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard (hunk 1 above)
Comment (Matthew-C) [inline BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard:21]: Xib does not support tvOS. We can not use UISwitch. All UI controls are
gone. Other things work just fine.

In BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard
https://github.com/ooyala/ios-sample-apps/pull/52#discussion_r44552568:

> ```
>                              <autoresizingMask key="autoresizingMask"/>
>                              <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="WCw-Qf-5nD" id="37f-cq-3Eg">
> ```
> -                                    <rect key="frame" x="0.0" y="0.0" width="320" height="43"/>
> -                                    <rect key="frame" x="0.0" y="0.0" width="600" height="43.5"/>

is this just an artifact of opening the BasicPlayback main.storyboard, or
did you actually modify something?

—
Reply to this email directly or view it on GitHub
https://github.com/ooyala/ios-sample-apps/pull/52/files#r44552568.

Comment (Matthew-C) [inline BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard:21]: This is just storyboard.

**Pull request merged**